pkg/kubelet/kubelet_pods.go

```diff
@@ -369,7 +402,8 @@ func (kl *Kubelet) GenerateRunContainerOptions(pod *v1.Pod, container *v1.Contai
 		return nil, false, err
 	}
 
-	opts.Mounts, err = makeMounts(pod, kl.getPodDir(pod.UID), container, hostname, hostDomainName, podIP, volumes)
+	mountPropagationEnabled := utilfeature.DefaultFeatureGate.Enabled(features.MountPropagation)
+	opts.Mounts, err = makeMounts(pod, kl.getPodDir(pod.UID), container, hostname, hostDomainName, podIP, volumes, mountPropagationEnabled)
```

Comment:
Why passing `mountPropagationEnabled` multiple times when instead, you could check that directly in `translateMountPropagation()`?

Reply:
Good question. Initially there was `kubelet --experimental-mount-propagation` that needed passing all over the place and this is the last remainder. Removed.